Dockerfile

```diff
@@ -11,17 +10,21 @@ ENV INSTALLED_SEMGREP_VERSION=0.52.0
 RUN apk add --no-cache --virtual=.build-deps build-base libffi-dev openssl-dev &&\
     apk add --no-cache --virtual=.run-deps bash git less libffi openssl &&\
     # Need to pin cryptography version to avoid Rust compiler dependency
-    pip install --no-cache-dir cryptography==3.3.2 poetry==1.1.6 &&\
-    pip install --no-cache-dir pipx &&\
+    pip install --no-cache-dir pipx~=0.16.1.0 &&\
     pipx install semgrep==${INSTALLED_SEMGREP_VERSION} &&\
+    (pip freeze | xargs pip uninstall -y) &&\
+    pip install --no-cache-dir cryptography==3.3.2 poetry==1.1.6 &&\
```

Comment:
We should probably update `cryptography` to the latest version at some point. I think the compiler dependency problems you mentioned above have been fixed in the latest versions, so long as you `pip install -U pip` first :+1: